### PR TITLE
LR-491 : ML Program UserInfoExhaust Data Product

### PR DESCRIPTION
**Pull request by Shakthieshwari**
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

<table>
<td>
  <a href="https://project-sunbird.atlassian.net/browse/LR-491" title="LR-491" target="_blank">StoryLR-491</a>  User detail report for programs - Data Product
  </td></table>
  <br />
 

### Type of change

Please choose appropriate options.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### How Has This Been Tested?

Please describe the tests that you ran to verify your changes in the below checkboxes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Ran Test A
- [ ] Ran Test B

**Test Configuration**:
* Software versions:
* Hardware versions:

### Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by Hari-stackroute — APPROVED** [review]

File: ansible/inventory/env/group_vars/all.yml [hunk 1]
```diff
@@ -78,7 +78,7 @@ single_node_es_host: "{{ groups['core-es'][0] }}"
 
 #Druid Proxy APi service
 sunbird_druid_broker_host: "http://{{ groups['raw-broker'][0] }}"
-sunbird_learner_service_url: "http://{{private_ingressgateway_ip}}/learner"
+sunbird_learner_service_url: "http://{{private_ingressgateway_ip}}/userorg"
```
Comment (AmiableAnil) [inline ansible/inventory/env/group_vars/all.yml]: Please revert this change. This change is required from release-5.4.0 onwards.

Comment (Shakthieshwari) [reply ansible/inventory/env/group_vars/all.yml]: @AmiableAnil But when i test with release 5.3 branch itself, it was failing. If i revert back this changes, If i deploy our forked repo with release-5.3.0 branch, will it work ?

Comment (Shakthieshwari) [reply ansible/inventory/env/group_vars/all.yml]: @AmiableAnil I have reverted the changes and pushed, Please review it and merge.
But please answer to my above query as well ..

Comment (AmiableAnil) [reply ansible/inventory/env/group_vars/all.yml]: The above change is part of release-5.4.0. And in lern dev env we have started deploying the release-5.4.0 changes. And your changes are planned for release-5.3.0, so release-5.4.0 changes should not go in release-5.3.0.

Comment (Shakthieshwari) [reply ansible/inventory/env/group_vars/all.yml]: @AmiableAnil Please let me know when to connect on this point ? Please do the needful at the earliest

**Review by AmiableAnil — APPROVED** [review]

**Review by AmiableAnil — APPROVED** [review]
